src/debug/HyprCtl.cpp

```diff
@@ -101,11 +128,13 @@ R"#({
                     ((int)w->m_bIsX11 == 1 ? "true" : "false"),
                     (w->m_bPinned ? "true" : "false"),
                     (w->m_bIsFullscreen ? "true" : "false"),
-                    (w->m_bIsFullscreen ? (g_pCompositor->getWorkspaceByID(w->m_iWorkspaceID) ? g_pCompositor->getWorkspaceByID(w->m_iWorkspaceID)->m_efFullscreenMode : 0) : 0)
+                    (w->m_bIsFullscreen ? (g_pCompositor->getWorkspaceByID(w->m_iWorkspaceID) ? g_pCompositor->getWorkspaceByID(w->m_iWorkspaceID)->m_efFullscreenMode : 0) : 0),
+                    (g_pLayoutManager->getCurrentLayout()->getLayoutName() == "dwindle" ? getGroupedData(w, format).c_str() : ""),
+                    (w->m_pSwallowed ? w->m_pSwallowed->getPID() : 0)
```

Comment:
using PID for swallowed kinda sucks. Why not use the handle? `w.get()` and then %x in `"swallowing"` above